### DISCUSSION TITLE
ESP32 board packages support.

### DISCUSSION THREAD

**Discussion by souravlal** [marcel-licence/esp32_basic_synth#109] [Q&A]
Hi, marcel-licence.
Does this ESP32 Synth project, ESP32 FM Synth project and the ESP32 MIDI Sampler project support the latest ESP32 boards version 2.0.11? If not, which version is supported for all the above-mentioned projects?
Because now, when you try to install ESP32 boards fresh in Arduino IDE the older versions are not there as an option to install. Only versions 2.0.11 and 2.0.10 is there as two options. I also don't know how to install older versions manually.
Your reply is much appreciated. Thanks.

**Comment by marcel-licence** [comment]
Hi @souravlal ,

in https://github.com/marcel-licence/esp32_basic_synth/blob/main/doc/board_info.md you will find the versions I've tested (only compilation). You could try other versions but I cannot say if it would work or not.
I think I should check to get a newer version soon (in case the latest version).
In the old Arduino version you could add some json links to find other packages. I think it would be too much work to go back to older versions to get the projects running. I think a support of newer versions might be possible.

Hope this helps. Best regards,

Marcel

**Reply by souravlal** [reply]
Thanks

**Reply by marcel-licence** [reply]
I tried to compile both with 2.0.13 with success. I don't know if the execution works. I faced some issues with the buffer initialization (PSRAM) in the past. Hope will work. Please let me know if you like

**Reply by souravlal** [reply]
I will give it a try.
